code/game/gamemodes/cult/cult_comms.dm

```diff
@@ -40,6 +40,8 @@
 	else
 		my_message = "<span class='cultspeech'><b>[(ishuman(user) ? "Acolyte" : "Construct")] [user]:</b> [message]</span>"
 	for(var/mob/M in GLOB.mob_list)
```

Comment:
Better way to fix it, make this `GLOB.player_list`, as that includes everyone, but excludes `/mob/new_player`

Reply:
Makes sense. Thanks!

Reply:
I want a clap reaction now just because I like how easy this solution is
👏